feat: add fish shell completions to README (#52)
dty1er/kubecolor: README.md

#### fish

> For fish, you can achieve completions for `kubecolor` in two steps:

Reviewer: Can you change this sentence?
I want to emphasize that this is unofficial hack because it is not official way provided by kubernetes project.

```suggestion
Fish completion is officially unsupported by kubectl, so it is unsupported by kubecolor as well.
However, there are 2 ways we can introduce to make completion work. 
Please keep in mind these ways are a kind of "hack" and not officially supported.

1. Use "fish-kubectl-completions" with kubecolor

[evanlucas/fish-kubectl-completions](https://github.com/evanlucas/fish-kubectl-completions) is popular for fish users who wants kubectl completions in fish shell.
You can "override" kubectl with kubecolor and make it work even with kubecolor. You need to follow below 2 steps:
```

> 1. Install `kubectl` completions (https://github.com/evanlucas/fish-kubectl-completions)
> 
>    ```shell
>    fisher install evanlucas/fish-kubectl-completions
>    ```
> 
> 2. Add the following function to your `config.fish` file:
> 
>    ```shell
>    function kubectl
>      kubecolor $argv
>    end
>    ```

Reviewer: Please add indents to them because they belong to above `1. Use "fish-kubectl-completions" with kubecolor`
```suggestion
  1. Install `kubectl` completions (https://github.com/evanlucas/fish-kubectl-completions)

   ```shell
   fisher install evanlucas/fish-kubectl-completions
   ```

  2. Add the following function to your `config.fish` file:

   ```shell
   function kubectl
     kubecolor $argv
   end
   ```
```

> **NOTE:** This will override `kubectl` command. If you wish to preserve both `kubectl` and `kubecolor` with completions, you need to copy completions for `kubecolor` command (e.g. https://github.com/awinecki/fish-kubecolor-completions).

Reviewer: I'm pretty sure some people will prefer fish-kubecolor-completions, so let's add some description:
```suggestion
2. Use awinecki/fish-kubecolor-completions

The first way will override `kubectl` command. If you wish to preserve both `kubectl` and `kubecolor` with completions, you need to copy evanlucas/fish-kubectl-completionsfor for `kubecolor` command.

For this purpose, you can use [awinecki/fish-kubecolor-completions](https://github.com/awinecki/fish-kubecolor-completions).
```
